eng/source-build-release-announcement.md

```diff
@@ -14,6 +14,6 @@ Please use the [$TAG tag]($TAG_URL) to source-build .NET version $RUNTIME_VERSIO
   "Channel": "$RELEASE_CHANNEL",
   "Runtime": "$RUNTIME_VERSION",
   "Sdk": "$SDK_VERSION",
-  "InstallerTag": "$TAG"
+  "RepoTag": "$TAG"
```

Comment:
This doesn't provide context of what repo the tag is for. I'm assuming `$TAG` is still just the tag name. But it needs to include the repo with it for it to be relevant.

Reply:
Would we add a new property `TagUrl`?